test/NinjaTools.FluentMockServer.API.Tests/Configuration/ConfigFileProviderTests.cs

```diff
@@ -72,13 +65,14 @@ public void Returns_ConfigurationFile_When_Only_One_Yaml_File()
             configurations[1].Configurations.Should().HaveCount(2);
             var setupB = configurations[1].Configurations.First();
             setupB.Action.Response.StatusCode.Should().Be(201);
-            setupB.Matcher.Path.Should().Be("/some/path");
-            setupB.Matcher.Method.Should().Be("POST");
+            setupB.Matcher.Path.ToPath().Should().Be("/some/path");
+            setupB.Matcher.Method.MethodString.Should().Be("POST");
 
 
             var setupC = configurations[1].Configurations.ElementAt(1);
-            setupC.Matcher.Headers.Should().HaveCount(2);
-            setupC.Matcher.Path.Should().Be("/");
+          //  Dump(setupC, "Setup - C");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4802644)